tests/wdio/slide-panel-manager/slide-panel-manager-spec.js

```diff
@@ -42,13 +42,13 @@ Terra.describeViewports('SlidePanelManager', ['large'], () => {
   });
 
   it('does not display a navigation prompt when attempting to go back to the nested panel', () => {
-    browser.click('[class*="SlidePanel"] [class*="ActionHeader"] .header-back-button');
+    browser.click('[class*="SlidePanel"] [class*="ActionHeader"] [class*="back-button"]');
     expect(browser.isExisting('[class*="notification-dialog"]')).to.be.false;
     browser.waitForVisible('[class*="slide-group"] #DemoContainer-2 .disclose-tiny');
   });
 
   it('displays a navigation prompt when attempting to go back to the root panel', () => {
-    browser.click('[class*="SlidePanel"] [class*="ActionHeader"] .header-back-button');
+    browser.click('[class*="SlidePanel"] [class*="ActionHeader"] [class*="back-button"]');
```

Comment:
@mjhenkes  you should use the new selector added recently: https://github.com/cerner/terra-core/pull/2862/files